src/Java.Interop/Java.Interop/JavaPrimitiveArrays.cs

```diff
@@ -246,7 +247,7 @@ public override IList<Boolean> CreateGenericValue (ref JniObjectReference refere
 						(ref JniObjectReference h, JniObjectReferenceOptions o) => new JavaBooleanArray (ref h, o));
 			}
 
-			public override JniValueMarshalerState CreateGenericObjectReferenceArgumentState (IList<Boolean> value, ParameterAttributes synchronize)
+			public override JniValueMarshalerState CreateGenericObjectReferenceArgumentState ([MaybeNull]IList<Boolean> value, ParameterAttributes synchronize)
```

Comment:
These changes should likewise prompt a change to [`src/Java.Interop/Java.Interop/JavaPrimitiveArrays.tt`](https://github.com/xamarin/java.interop/blob/master/src/Java.Interop/Java.Interop/JavaPrimitiveArrays.tt).